enhancements/storage/csi-driver-operator-merge.md

```diff
@@ -469,6 +469,80 @@ N/A
 
 Same as today.
 
+## Process of moving operators to csi-operator monorepo
+
+We have come with following flow for moving operators from their own repository into csi-operator mono repo.
+
+### Avoid .gitignore related footguns
+
+Remove any .gitignore entries in the source repository that would match a directory / file that we need. For example, `azure-disk-csi-driver-operator` in .gitignore matched `cmd/azure-disk-csi-driver-operator` directory that we really need not to be ignored. See https://github.com/openshift/csi-operator/pull/110, where we had to fix after merge to csi-operator.
+
+### Move existing code into csi-operator repository
+
+Using git-subtree move your existing operator code to https://github.com/openshift/csi-operator/tree/master/legacy
+
+```
+git subtree add --prefix legacy/azure-disk-csi-driver-operator https://github.com/openshift/azure-disk-csi-driver-operator.git master --squash
+git subtree push --prefix legacy/azure-disk-csi-driver-operator https://github.com/openshift/azure-disk-csi-driver-operator.git master
+```
+
+### Add Dockerfiles for building images from new location
+
+Place a `Dockerfile.<operator>` and `Dockerfile.<operator>.test` at top of csi-operator tree and make sure that you are able to build an image of the operator from csi-operator repository.
```

Comment:
We consolidated the various `Dockerfile.<operator>.test` files into a single `Dockerfile.test`, there generally shouldn't be any need to add a new one unless some operator is doing more than `FROM src`.

Reply:
fixed.